tests/kernel/qa_test_klp.pm

```diff
@@ -30,9 +32,9 @@ sub run {
 
     zypper_call('in -l git gcc kernel-devel');
 
-    assert_script_run('git clone ' . get_var('QA_TEST_KLP_REPO'));
+    assert_script_run('git clone ' . $git_repo);
 
-    assert_script_run('cd qa_test_klp;bats klp.bats');
+    assert_script_run("cd qa_test_$test_type;bats $test_type.bats", 2760);
```

Comment:
Do we really need 46 minutes timeout here? Look like previously it worked with default one which is 30 seconds. Otherwise LGTM.

Reply:
@rwx788  because now correctly creates high load during testsuite run :) , Value was taken from original CTCS2 kgraft testsuite.